### PR TITLE
Small improvements to 101 tutorial.

### PR DESCRIPTION
**Pull request by joerg84**
## What are your changes?
Changed order of components and Terminology as the former makes use of the second.

## What type of changes does your doc introduce?
- [x] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [x] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [x] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

## If you included a comment with your commit, it appears here:

### CODE REVIEW

File: 1.8/usage/tutorials/dcos-101/cli.md [hunk 1]
```diff
@@ -36,19 +36,19 @@ We will make further use of the CLI in the sections that follow.
...
-A description of all components can be found in the [documentation](https://dcos.io/docs/1.8/overview/components/).
-* Marathon starts and monitors DC/OS applications and services.
-* Mesos is the kernel of DC/OS and responsible for low-level task maintenance.
-* Mesos DNS provides service discovery within the cluster.
-* Minuteman is the internal layer 4 load balancer.
-* Admin Router is an open source NGINX configuration that provides central authentication and proxy to DC/OS services.
-* Universe is the package repository that holds the DC/OS services (e.g., Apache Spark of Apache Cassandra) that you can install on your cluster directly from the DC/OS UI and CLI.
-
 ## DC/OS Terminology
 Again we will focus on the most important terminology here. More details can be found in the [documentation](https://dcos.io/docs/1.8/overview/architecture/).
```
Comment (judithpatudith) [inline 1.8/usage/tutorials/dcos-101/cli.md]: Remove "Again" Capitalize "W" in we.

**Comment by judithpatudith** [comment]
+1 Looks good with comments.

**Comment by joerg84** [comment]
Addressed comment.

File: 1.8/usage/tutorials/dcos-101/cli.md [hunk 2]
```diff
@@ -36,19 +36,19 @@ We will make further use of the CLI in the sections that follow.
 We have already encountered several DC/OS components (including Mesos, Marathon, or Metronome) while experimenting with the DC/OS CLI.
 But what other components make up DC/OS?
 
+## DC/OS Terminology
+We will focus on the most important terminology here. More details can be found in the [documentation](https://dcos.io/docs/1.8/overview/architecture/).
+* Master: Aggregates resource offers from all agent nodes and provides them to registered frameworks.
```
Comment (joel-hamill) [inline 1.8/usage/tutorials/dcos-101/cli.md]: This is a bit different than https://dcos.io/docs/1.8/overview/concepts/#dcos-master-node.

File: 1.8/usage/tutorials/dcos-101/cli.md [hunk 3]
```diff
@@ -36,19 +36,19 @@ We will make further use of the CLI in the sections that follow.
...
+* Agent: Runs a discrete Mesos task on behalf of a framework. The synonym of agent node is worker or slave node.
+* Scheduler: The scheduler component of a service, for example, the Marathon scheduler.
```
Comment (joel-hamill) [inline 1.8/usage/tutorials/dcos-101/cli.md]: This is a bit different than: https://dcos.io/docs/1.8/overview/concepts/#-a-name-dcos-scheduler-a-scheduler.

**Comment by stbof** [comment]
Will revisit in newer versions in new doc site.

**Pull request closed without merging**